workloads/kube-burner/run.sh

```diff
@@ -15,8 +15,10 @@ case ${WORKLOAD} in
   ;;
   node-density)
     WORKLOAD_TEMPLATE=workloads/node-pod-density/node-pod-density.yml
+    INDEXING="false"
+    POD_NODE_SELECTOR=${POD_NODE_SELECTOR:-'{kubernetes.io/os: windows}'}
```

Comment:
You're setting these variables by default regardless you want to trigger the test against windows workers or not. That would affect the current behavior with Linux nodes.

Reply:
Yes, you're right. That was a change for my local testing I can remove.

Reply:
removed it now